server/odc-service/src/main/java/com/oceanbase/odc/config/BeanConfiguration.java

```diff
@@ -171,4 +182,23 @@ public OBDatabaseUser getSysTenantUser(String instanceId) {
         }
     }
 
+    /**
+     * In order to adapt to the fact that there is no ODP_SHARDING_OB_MYSQL in DialectType of the
+     * front-end, ODP_SHARDING_OB_MYSQL is converted to OB_MYSQL during serialization.
+     */
+    private static class DialectTypeOutputSerializer extends JsonSerializer<DialectType> {
+
+        @Override
+        public void serialize(DialectType dialectType, JsonGenerator jsonGenerator,
+                SerializerProvider serializerProvider) throws IOException {
+            if (Objects.nonNull(dialectType)) {
```

Comment:
what if null?

Reply:
Added logic for handling null ：`jsonGenerator.writeNull();`